meshapp/src/home/logged-out/home.tsx

```diff
@@ -1,16 +1,29 @@
 import { Grid, Theme, ThemeProvider, createTheme } from "@mui/material";
-import React from "react";
+import React, { useState } from "react";
 
 import loggedOutNav from "../../components/home-logged-out/loggedOutNavBar";
 import welcomeMessage from "../../components/home-logged-out/loggedOutWelcome";
 import advertSection from "../../components/home-logged-out/advertSection";
 import reviewsSection from "../../components/home-logged-out/reviewSection";
 import LoginInput from "../../components/login-form";
 import { deepmerge } from "@mui/utils";
+import { Login } from "@mui/icons-material";
+import ForgotPassword from "../../components/password-forms/initiate-reset-form";
 
 export default function LoggedOutPage() {
   const pageTheme = PageTheme();
-
+  
+  const [showForgotPassword, setShowForgotPassword] = useState(false);
+
+  const updateShowForgotPasswordState = () => {
+    if (showForgotPassword == false) {
+      setShowForgotPassword(true)
+    }
+    if (showForgotPassword == true) {
+      setShowForgotPassword(false)
+    }
+  }
```

Comment:
Looks like I forgot to comment the same thing here. Why was this code duplicated though? Can't the logic be moved out into a common function both files can use?

Reply:
https://github.com/LetsMesh/Site/issues/269 created an issue to track this